Index: Product.java
===================================================================
diff --git a/Product.java b/Product.java
deleted file mode 100644
--- a/Product.java	(revision e440585bbe12d83d892b82c55ee9247b4c7fc67d)
+++ /dev/null	(revision e440585bbe12d83d892b82c55ee9247b4c7fc67d)
@@ -1,16 +0,0 @@
-public class Product {
-
-    String productName, productType;
-    int quantity, productPrice;
-
-    public Product(String productName, String productType, int quantity, int productPrice) {
-        this.productName = productName;
-        this.productType = productType;
-        this.quantity = quantity;
-        this.productPrice = productPrice;
-
-    }
-
-
-}
-
